src/components/Skeleton/index.tsx

```diff
@@ -44,44 +43,16 @@ const Container = ({ children }: ContainerProps) => (
 );
 
 const Skeleton = ({ children }: SkeletonProps) => {
-  const {
-    id: userId,
-    agreeOnTermsOfService: isAgreeOnTermsOfService,
-    deviceType,
-  } = useValueRecoilState("loginInfo") || {};
+  const { id: userId, agreeOnTermsOfService: isAgreeOnTermsOfService } =
+    useValueRecoilState("loginInfo") || {};
   const error = useRecoilValue(errorAtom);
   const isLoading = userId === null;
-
-  const location = useLocation();
-  const { pathname } = location;
-
-  const [cookies, setCookies] = useCookies(["isOpposeSuggestApp"]);
-  const isOpposeSuggestApp = !!cookies?.isOpposeSuggestApp;
+  const { deviceType } = useValueRecoilState("loginInfo") || {};
```

Comment:
```suggestion
  const {
    id: userId,
    agreeOnTermsOfService: isAgreeOnTermsOfService,
    deviceType,
  } = useValueRecoilState("loginInfo") || {};
```

```
  const { id: userId, agreeOnTermsOfService: isAgreeOnTermsOfService } =
    useValueRecoilState("loginInfo") || {};
``` 
과
```
  const { deviceType } = useValueRecoilState("loginInfo") || {};
```
가 분리되어 있지 않아도 될 것 같아요

Reply:
ㅋㅎㅋㅎㅋㅎㅋㅎㅋ 제가 왜 그랬을까요ㅜㅠㅠ